nessie/sql_templates/create_intermediate_schema.template.sql

```diff
@@ -81,6 +81,10 @@ AS (
             en.enrollment_status = 'D'
             OR en.grade = 'W'
         )
+    /* Clear out duplicates. */
+    GROUP BY
```

Comment:
@raydavis - Would we need a having clause?

Reply:
Not that I can see off-hand, @sandeepmjay -- please say more!

Reply:
Looks like this was a filtering query. Thanks for the explanation @raydavis.